src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -218,37 +218,37 @@ private void SetGlobalErrorIfTrue(bool arg)
                 }
             }
 
-            void addCircularStructDiagnostics(NamespaceOrTypeSymbol symbol)
+            void addAfterAccessorBindingDiagnostics(Symbol symbol)
             {
-                if (symbol is SourceMemberContainerTypeSymbol sourceMemberContainerTypeSymbol && PassesFilter(filterOpt, symbol))
+                if (PassesFilter(filterOpt, symbol))
                 {
-                    _ = sourceMemberContainerTypeSymbol.KnownCircularStruct;
+                    symbol.AfterAccessorBindingChecks();
```

Comment:
I'm not sure I'm understanding correctly. I did 68ac2ac, but it might not be what you're thinking of. If that's the case, can you elaborate a bit more?